### DISCUSSION TITLE
Is it possible to change your primary email after you signed up for github global campus?

### DISCUSSION THREAD

**Discussion by undrscre** [community/community#88627] [GitHub Education] [answered]
### Hi everyone,

Hello! I recently signed up for the GitHub Student Developer pack, and I realized I just used the wrong email address to sign up for it. But my application is already approved! (Although my benefits aren't given yet)
Is it possible to change the email and reflect it throughout the benefits?

Sorry if I don't make sense, I am not a native speaker
Thanks!

**Comment by Ashishdheer** [comment, marked as answer]
First log in to your GitHub account, go to "Settings," click on "Email" in the left sidebar, and add your correct email address. Verify the new email by following the instructions sent to your inbox and set it as the primary email in the GitHub settings. This ensures that you receive all future communications and benefits associated with the GitHub Student Developer Pack at the correct email address. If you encounter any issues, consider reaching out to GitHub Support for assistance. Hope this will help

**Reply by mishmanners** [reply]
Make sure you keep your student email on file too to avoid any issues with access to the Student Developer Pack.
